packages/legend-application-studio/src/components/editor/edit-panel/connection-editor/RelationalDatabaseConnectionEditor.tsx

```diff
@@ -1151,6 +1135,218 @@ const RelationalConnectionStoreEditor = observer(
+                        <MenuContentItem
+                          key={postProcessorType.value}
+                          onClick={addPostProcessor(postProcessorType.value)}
+                        >
+                          New {postProcessorType.label} Post Processor
+                        </MenuContentItem>
+                      ))}
+                      menuProps={{
+                        anchorOrigin: {
+                          vertical: 'bottom',
+                          horizontal: 'right',
+                        },
+                        transformOrigin: {
+                          vertical: 'top',
+                          horizontal: 'right',
+                        },
+                        elevation: 7,
+                      }}
+                    >
+                      <PanelHeader title="post-processor">
+                        <PanelHeaderActionItem tip="Create Post-Processor">
+                          <PlusIcon />
+                        </PanelHeaderActionItem>
+                      </PanelHeader>
+                    </DropdownMenu>
+                    <div className="panel__content">
+                      {postProcessors.map((postProcessor, idx) => (
+                        <ContextMenu
+                          key={postProcessor._UUID}
+                          disabled={false}
+                          content={
+                            <MenuContent>
+                              <MenuContentItem
+                                onClick={deletePostProcessor(postProcessor)}
+                              >
+                                Delete
+                              </MenuContentItem>
+                            </MenuContent>
+                          }
+                          menuProps={{ elevation: 7 }}
+                        >
+                          <PanelExplorerItem
+                            title={`Post-Processor ${idx + 1}`}
+                            onSelect={() => selectPostProcessor(postProcessor)}
+                            isSelected={
+                              postProcessor ===
+                              postProcessorState?.postProcessor
+                            }
+                          />
+                        </ContextMenu>
+                      ))}
+                    </div>
+                  </div>
+                </ResizablePanel>
+                <ResizablePanelSplitter />
+
+                <ResizablePanel>
+                  {postProcessorState?.postProcessor &&
+                    renderEditorPostProcessor(
+                      connectionValueState,
+                      postProcessorState.postProcessor,
+                      plugins,
+                    )}
+                  {!postProcessorState && (
+                    <BlankPanelContent>
+                      {!postProcessors.length
+                        ? 'Add a post-processor to view properties'
```

Comment:
--> `Add a post-processor`, and `Select a post-processor to view`.

Also, have you accounted for `read-only` case? See my note above